http/base/src/main/java/org/wildfly/security/http/HttpConstants.java

```diff
@@ -187,4 +205,9 @@ private HttpConstants() {
     public static final String HTTP = "http";
     public static final String HTTPS = "https";
 
+    /**
+     * Bearer token pattern.
+     */
+    public static final Pattern BEARER_TOKEN_PATTERN = Pattern.compile("^Bearer *([^ ]+) *$", Pattern.CASE_INSENSITIVE);
```

Comment:
I know it is only a fairly simple pattern but could be useful to add to the comment the intent for the pattern so anyone debugging issues later can cross check.

Reply:
Good point, added.